src/components/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import Nav from './Nav/Nav';
```

Comment:
Was wondering why the App.js file is in the component folder? I feel like it might be better to put the App.js file in the top most layer (src without being nested in components) b/c I feel like App.js is just rendering components, and isn't a component itself.

Reply:
Good point, I had thought of it more as a parent component for everything else. But either way I think I wasn't super consistent with my organizing and I agree that your suggestion would be much cleaner. Thanks!